PROJECT

```diff
@@ -1,10 +1,10 @@
-domain: coreos.com
+domain: operators.coreos.com
 layout: go.kubebuilder.io/v3
 projectName: service-binding-operator
 repo: github.com/redhat-developer/service-binding-operator
 resources:
 - crdVersion: v1
-  group: operators
+  group: binding
```

Comment:
Previously the group name was plural `operators`, shouldn't the new group name be plural as well? 
`bindings` instead of `binding`.

Reply:
The previous group name was set so that it matches the group used by OLM. Best practice is to keep group name as singular, because they are very much equivalent to package names:

* https://cluster-api.sigs.k8s.io/developer/providers/implementers-guide/naming.html
* https://softwareengineering.stackexchange.com/questions/75919/should-package-names-be-singular-or-plural